core/src/scene/sceneLoader.cpp

```diff
@@ -66,6 +73,261 @@ glm::vec3 parseVec3(const Node& node) {
     return vec;
 }
 
+void SceneLoader::loadShaderConfig(YAML::Node shaders, ShaderProgram& shader) {
+
+    if (!shaders) {
+        return;
+    }
+
+    Node definesNode = shaders["defines"];
+    if (definesNode) {
+        for (const auto& define : definesNode) {
+            std::string name = define.first.as<std::string>();
+            std::string value = define.second.as<std::string>();
+            shader.addSourceBlock("defines", "#define " + name + " " + value);
```

Comment:
Update: they don't. 

I guess for now we have to deal with not being able to #define booleans.